contracts/MockContract.sol

```diff
@@ -82,6 +82,7 @@ contract MockContract is MockInterface {
 	bytes public constant MOCKS_LIST_END = "0xff";
 	bytes32 public constant MOCKS_LIST_END_HASH = keccak256(MOCKS_LIST_END);
 	bytes4 public constant SENTINEL_ANY_MOCKS = hex"01";
+    bytes public constant DEFAULT_FALLBACK_VALUE = abi.encode(false);
```

Comment:
indentation?

Reply:
```suggestion
      bytes public constant DEFAULT_FALLBACK_VALUE = abi.encode(false);
```

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2FVBVY9IJKDxwHK%2Fgiphy.gif&token=eagle" data-gitmeme-token="eagle"><img src="https://media2.giphy.com/media/VBVY9IJKDxwHK/giphy.gif" title="Created by gitme.me with /eagle"/></a>

Reply:
You've got the best Jiffs

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2Faaw0ay7NqrucU%2Fgiphy-downsized-medium.gif&token=jif" data-gitmeme-token="jif"><img src="https://media3.giphy.com/media/aaw0ay7NqrucU/giphy-downsized-medium.gif" title="Created by gitme.me with /jif"/></a>